config/projects.yml

```diff
@@ -80,6 +80,13 @@ classification-bundle:
         symfony: ['2.3', '2.7', '2.8', '3.0', '3.1']
         sonata_admin: ['3']
 
+classification-media-bundle:
+  branches:
+    master:
+      php: ['5.3', '5.4', '5.5', '5.6', '7.0']
```

Comment:
> Oh you mean code here will be coming from media / classification, and a BC solution should be provided?

That's the idea. I'll try to make a BC version for the stable classification bundle, so we need the lower deps for PHP. 

Maybe we could create a 0.x release for the new bundle for a short time. So we can drop it later with the 1.0 release.

> Also, why is there only a definition for the master branch?

Because it's the initial setup. We should we create two branches for this?